packages/office-ui-fabric-react/src/components/SearchBox/examples/SearchBox.Disabled.Example.tsx

```diff
@@ -1,12 +1,18 @@
 import * as React from 'react';
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
-import './SearchBox.Examples.scss';
+import { mergeStyleSets } from '@uifabric/styling';
+
+const styles = mergeStyleSets({
+  searchBox: {
```

Comment:
The new styles aren't doing quite the same thing. This is the old version, which applies the margin to **any SearchBox inside the example** (not the example root itself).
```
  .ms-SearchBoxExample .ms-SearchBox {	
    margin: 0 0 10px 0;
  }
```

Typically what we do now for cases like this is rather than using `mergeStyleSets`, replace the wrapper `div` with a `Stack` that has `childrenGap` set (I think it should be 20 instead of 10 to look less cramped). Basically like this:

```
import { Stack } from 'office-ui-fabric-react/lib/Stack';

const stackTokens = { childrenGap: 20 };

...
return (
  <Stack tokens={stackTokens}>
    <SearchBox
...
```

Reply:
That makes sense. I updated it to your spec